registry/storage/driver/rados/rados.go

```diff
@@ -471,6 +471,7 @@ func (d *driver) Delete(ctx context.Context, objectPath string) error {
 		totalSize, err := d.getXattrTotalSize(ctx, oid)
 
 		if err != nil {
+			d.deleteOid(objectPath)
```

Comment:
This error still needs to be checked and at least reported:

```
if err := d.deleteOid(objectPath); err != nil {
    log.Println("error deleting object: %v", err)
}
```

Something to the effect of the above should work. You may want to be a little more creative with the error message to provide move context about the failure.